frontend/src/common/components/SearchBar.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { TextInput } from 'react-native';
+import PropTypes from 'prop-types';
+
+const defaultStyle = {
+  fontSize: 24,
+};
+
+function SearchBar({ style, onSubmit, placeholder }) {
+  const [val, setVal] = useState('');
```

Comment:
```suggestion
  const [val, setVal] = useState('');
  
```
for better readability add a line